src/Microsoft.Tye.Core/ConfigModel/ConfigService.cs

```diff
@@ -4,13 +4,19 @@
 
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using Tye;
 using YamlDotNet.Serialization;
 
 namespace Microsoft.Tye.ConfigModel
 {
     public class ConfigService
     {
+        const string errorMessage = "a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?')";
```

Comment:
```suggestion
        const string errorMessage = "A service name must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?')";
```

Reply:
Let's make it clearer that it's about service names. I don't expect most users to know what a DNS label is :)

Reply:
This change appears to have been reverted. It's back to saying `a DNS-1035 label...`